File: skin/sample/src/main/java/com/nqmgaming/cat/SkinApp.java
package com.nqmgaming.cat;

import android.app.Application;

import timber.log.Timber;

public class SkinApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: skin_sample/src/main/java/com/nqmgaming/skin_sample/SkinApp.kt
package com.nqmgaming.skin_sample

import android.app.Application
import timber.log.Timber

class SkinApp : Application() {
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }
}
